src/main/java/org/gridsuite/study/server/NetworkModificationTreeService.java

```diff
@@ -497,9 +504,9 @@ public BuildInfos getBuildInfos(UUID nodeUuid) {
         return buildInfos;
     }
 
-    private void fillInvalidateNodeInfos(NodeEntity node, InvalidateNodeInfos invalidateNodeInfos, boolean removeOnlyResults) {
+    private void fillInvalidateNodeInfos(NodeEntity node, InvalidateNodeInfos invalidateNodeInfos, boolean invalidateOnlyChildrenBuildStatus) {
 
-        if (!removeOnlyResults) {
+        if (!invalidateOnlyChildrenBuildStatus) {
```

Comment:
well we invalidate securityAnalysisResults as well then `invalidateOnlyChildrenBuildStatus` seems a bad naming...

Reply:
We do not invalidate `securityAnalysisResults`
We fill  `invalidateNodeInfos`